tests/e2e_tests/subcommands/subnet/test_list.py

```diff
@@ -0,0 +1,29 @@
+import bittensor
+from bittensor.commands import RegisterSubnetworkCommand
+from tests.e2e_tests.utils import setup_wallet
+
+"""
+Test the list command before and after registering subnets. 
+
+Verify that:
+* list of subnets gets displayed
+-------------------------
+* Register a subnets
+* Ensure is visible in list cmd
+"""
+
+
+def test_list_command(local_chain, capsys):
+    # Register root as Alice
+    keypair, exec_command, wallet = setup_wallet("//Alice")
+
+    subtensor = bittensor.subtensor(network="ws://localhost:9945")
```

Comment:
Let's make use of the `local_chain` fixture rather than hard-coding this value, for standardisation.